### DISCUSSION TITLE
Performance CarLinkit device vs Raspberry Pi 4?

### DISCUSSION THREAD

**Discussion by mrdance** [tesla-android/issue-tracker#286] [General]
I am looking at this solution compared to device like https://carlinkitcarplay.com/. I can assume that Raspberry Pi 4 is faster but can anyone give any feedback and tell me if there are any other pros/cons.

Mainly interested in the wireless Android Auto.

**Comment by dandelionclock** [comment]
You also get to have whatever android app you want running on your tesla screen. That includes youtube while driving (not recommended)

**Comment by mrdance** [comment]
Ok, so Tesla android is not just Android auto but full Android?

**Reply by josbmw** [reply]
That is correct. You can use Android Auto, Apple Carplay or native Android. Unfortunately, in the current version of TA there is no location info. But it should be available in the next release, so you can run Waze or ABRP native on Tesla Android.
